src/ar.android.ts

```diff
@@ -15,6 +17,16 @@ declare const com, android, global, java: any;
 let _fragment;
 let _origin;
 
+const addNode = (options: ARAddOptions, parentNode: SCNNode): Promise<ARGroup> => {
```

Comment:
`SCNNode` is an iOS class, I think this is a copy-paste error 😉

Reply:
yup. nice catch